Resources/doc/reference/action_create_edit.rst

```diff
@@ -33,13 +33,13 @@ When adding a form group to your edit/create form, you may specify some options
 - ``description``: to complete
 - ``translation_domain``: to complete
 
-To specify options, do as follow:
+To specify options, do as follows:
```

Comment:
@GromNaN, well, this looks cleaner, but both options look grammatically correct to me.